inlong-tubemq/tubemq-client-twins/tubemq-client-cpp/example/producer/test_producer.cc

```diff
@@ -0,0 +1,156 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <stdio.h>
+#include <string>
+#include <set>
+#include <thread>
+#include <chrono>
+#include <iostream>
+
+#include "tubemq/tubemq_client.h"
+#include "tubemq/tubemq_config.h"
+#include "tubemq/tubemq_errcode.h"
+#include "tubemq/tubemq_message.h"
+#include "tubemq/tubemq_return.h"
+#include "utils.h"
+
+using namespace std;
+using namespace tubemq;
+
+using tubemq::ProducerConfig;
+using tubemq::TubeMQProducer;
+
+struct MessageSentCallback {  
+  static AtomicLong TOTAL_COUNTER;
+  static AtomicLong SENT_SUCC_COUNTER;
+  static AtomicLong SENT_FAIL_COUNTER;
+  static AtomicLong SENT_EXCEPT_COUNTER;
+  
+  void operator()(const ErrorCode& error_code) {
+    TOTAL_COUNTER.IncrementAndGet();
+    if (error_code.Value() == err_code::kErrSuccess) {
+      SENT_SUCC_COUNTER.IncrementAndGet();
+    } else {
+      SENT_FAIL_COUNTER.IncrementAndGet();
+    }
+	}
+
+  static void ShowSentResult() {
+    std::cout << "Finished, total sent: " << TOTAL_COUNTER.Get() << ", sent successfully: " << SENT_SUCC_COUNTER.Get() << std::endl;
+  }
+};
+
+AtomicLong MessageSentCallback::TOTAL_COUNTER;
+AtomicLong MessageSentCallback::SENT_SUCC_COUNTER;
+AtomicLong MessageSentCallback::SENT_FAIL_COUNTER;
+AtomicLong MessageSentCallback::SENT_EXCEPT_COUNTER;
+
+const uint64_t MSG_COUNT = 100;
+bool SYNC_PRODUCTION = false;
+const uint32_t MSG_DATA_SIZE = 2048;
+
+int main(int argc, char* argv[]) {
+	if (argc < 3) {
```

Comment:
You need use .clang-format your code. clang-format file in this : 
https://github.com/apache/inlong/blob/master/inlong-tubemq/tubemq-client-twins/tubemq-client-cpp/.clang-format

Reply:
I have formatted the code style using clang-format [ca9de3](https://github.com/apache/inlong/pull/6612/commits/ca9dec3f049b912386c7588461dbc39e7f7852b6)